src/main/java/frc/robot/subsystems/BallPickup.java

```diff
@@ -0,0 +1,104 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// Java from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+package frc.robot.subsystems;
+
+import frc.robot.Robot;
+import frc.robot.commands.*;
+import edu.wpi.first.wpilibj.livewindow.LiveWindow;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.PIDOutput;
+import edu.wpi.first.wpilibj.PIDSource;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+
+
+/**
+ *
+ */
+public class BallPickup extends Subsystem {
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+private CANSparkMax geckoMotor;
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+private DoubleSolenoid pivotPiston;
+private Compressor compressor;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+
+    public BallPickup() {
+        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+pivotPiston = new DoubleSolenoid(0, 2, 3);
```

Comment:
Formatting issues, indent the code to match the braces
Remove unnecessary whitespaces